tests/lib/bundle_cli_test.py

```diff
@@ -0,0 +1,16 @@
+import unittest
+from codalab.lib.bundle_cli import BundleCLI
+
+
+class BundleCliTest(unittest.TestCase):
+    def setUp(self) -> None:
+        self.bundle_cli = BundleCLI
+        self.argv = ['cl', 'run', '---', 'echo', '']
```

Comment:
Why is this part of `self`?  It's likely that we'll have multiple tests with different `argv`s.